src/main/java/uk/gov/companieshouse/pscfiling/api/service/CompanyProfileService.java

```diff
@@ -4,6 +4,9 @@
 import uk.gov.companieshouse.api.model.transaction.Transaction;
 import uk.gov.companieshouse.pscfiling.api.exception.CompanyProfileServiceException;
 
+/**
+ * The company profile service layer.
+ */
```

Comment:
perhaps to add that it retrieves company profile data from Company Profile API or something?